actions/commitlint/commitlint.test.ts

```diff
@@ -4,8 +4,8 @@ import { TrunkActionDriver } from "tests/driver";
 
 const preCheck = (driver: TrunkActionDriver) => {
   driver.writeFile(
-    "commitlint.config.js",
-    "module.exports = {extends: ['@commitlint/config-conventional']}",
+    "commitlint.config.mjs",
+    "export default {extends: ['@commitlint/config-conventional']}",
```

Comment:
Not sure how renaming the file and using `export default` should be handled by trunk when upgrading to this version. I.e. how to handle breaking changers in tools supported by trunk?

Reply:
I suppose its up to the user to configure the commitlint config file. I’ll need to update the commitlint readme here: https://github.com/trunk-io/plugins/tree/main/actions/commitlint

Reply:
Thanks for the PR @rriski and for bringing this to our attention! I'll speak with the team about if there is any better way we can handle the update process for users. I think we will probably want to go ahead and land this, and also update the readme instructions, as you mentioned, but I'll follow-up again shortly.